docs/api/menu-item.md

```diff
@@ -147,15 +147,15 @@ item's submenu, if present.
 
 #### `menuItem.type`
 
-A `String` indicating the type of the item.
+A `String` indicating the type of the item. Can be `normal`, `separator`, `submenu`, `checkbox` or `radio`.
```

Comment:
Oxford comma?
```suggestion
A `String` indicating the type of the item. Can be `normal`, `separator`, `submenu`, `checkbox`, or `radio`.
```

Reply:
I don't think this will play nicely with the parser

Reply:
That's unfortunate.